src/main/resources/META-INF/persistence.xml

```diff
@@ -6,11 +6,6 @@
     
     	<provider>org.hibernate.ejb.HibernatePersistence</provider>
     	<class>com.car.app.model.Car</class>
-    	<exclude-unlisted-classes>false</exclude-unlisted-classes>
-    	<!-- enable cache 
-    	DISABLE_SELECTIVE: To cache only the entities annotated with @Cacheable or @Cacheable(true)  -->
-    	<shared-cache-mode>ENABLE_SELECTIVE</shared-cache-mode>
-    	
     	<properties>
 			<property name="javax.persistence.jdbc.driver" value="org.postgresql.Driver" />	        
 	        <property name="javax.persistence.jdbc.url" value="jdbc:postgresql://localhost:5432/auto-car-db" />
```

Comment:
Instead of using a url / user / password here, reference a datasource like here:
https://thorben-janssen.com/jpa-persistence-xml/#Reference_a_datasource